server/odc-core/src/main/java/com/oceanbase/odc/core/migrate/Migrates.java

```diff
@@ -85,7 +88,15 @@ public void migrate() {
                 version2Migratables.keySet().stream().map(Version::new).sorted().collect(Collectors.toList());
         log.info("versionCount={}", sortedVersions.size());
 
-        degradeCheck(sortedVersions.get(sortedVersions.size() - 1).getVersion());
+        String currentVersion = sortedVersions.get(sortedVersions.size() - 1).getVersion();
+
+        if (VersionUtils.isLessThan(configuration.getInitVersion(), "4.3.0.0")) {
```

Comment:
This logic should be moved to the apsara implementation.

Reply:
got it, done.